crates/chia-tools/src/bin/analyze-chain.rs

```diff
@@ -161,9 +162,14 @@ fn main() {
         let start_conditions = SystemTime::now();
         // we pass in what's left of max_cost here, to fail early in case the
         // cost of a condition brings us over the cost limit
-        let Ok(conds) =
-            parse_spends::<MempoolVisitor>(&a, generator_output, ti.cost - clvm_cost, MEMPOOL_MODE)
-        else {
+        // TODO: Use real constants
```

Comment:
Perhaps we should port the real constants over as well. Out of scope for this PR though, so feel free to resolve this comment

Reply:
yeah, at some point we should. I want to defer additional complexity from that though.